h/static/scripts/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "lib": ["es2018", "dom"],
```

Comment:
This can safely be bumped to `es2020` at least.

Reply:
Done: https://github.com/hypothesis/h/pull/8789/commits/7633256280be0e55e3a4df5111c264a25f1c574f